File: URI/Java/Lanche.java
package Java;

import java.util.Scanner;
// import java.lang.Math;

public class Lanche {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int qte, tag = input.nextInt();
        float preco = 0;
        qte = input.nextInt();

        switch (tag) {
            case 1:
                preco = 4.00f;
                break;
            case 2:
                preco = 4.50f;
                break;
            case 3:
                preco = 5.00f;
                break;
            case 4:
                preco = 2.00f;
                break;
            case 5:
                preco = 1.50f;
                break;
        }
        String str = String.format("%.2f", preco*qte);
        System.out.println("Total: R$ "+str);
        input.close();
    }
}

File: URI/Python/Lanche.py
infos = [int(x) for x in input().split()]
traduor = {
    1 : 4.0,
    2 : 4.5,
    3 : 5.0,
    4 : 2.0,
    5 : 1.5 
}
print("Total: R$ {:.2f}".format(traduor[infos[0]]*infos[1]))